src/components/Conversion/ConversionNOB.vue

```diff
@@ -15,7 +15,7 @@
       </v-col>
 
       <v-col cols="12" sm="9">
-        <div v-if="onEditMode">
+        <template v-if="onEditMode">
```

Comment:
Ideally this component would use rows and cols for layout (like the other components) but I didn't want to spend all day refactoring this so I left almost everything alone and just tweaked some properties to align everything.